app/tasks/doctor.php

```diff
@@ -135,8 +139,15 @@
 
         try {
             $mail = $register->get('smtp'); /* @var $mail \PHPMailer\PHPMailer\PHPMailer */
+            if ($receiver != '') {
+                $mail->addAddress($receiver);
+            } elseif ($interactive == "Y" || $interactive == "y") {
```

Comment:
Let's convert interactive to lower case and keep one condition check
```suggestion
            } elseif (strtolower($interactive === "y")) {
```

Reply:
Thanks, looks like I really need to read up a bit on php....